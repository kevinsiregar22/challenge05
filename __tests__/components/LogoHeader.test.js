import React from 'react';
import LogoHeader from '../../src/components/LogoHeader';
import {render} from '@testing-library/react-native';

describe('should LogoHeader', () => {
  test('test LogoHeader', () => {
    const component = render(<LogoHeader />);
    expect(component).toMatchSnapshot();
  });
});
