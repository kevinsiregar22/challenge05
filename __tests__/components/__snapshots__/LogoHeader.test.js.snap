// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should LogoHeader test LogoHeader 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../src/assets/icons/book6.png",
      }
    }
    style={
      Object {
        "height": 235.71428571428572,
        "width": 235.71428571428572,
      }
    }
  />
</View>
`;
